session/database.rst

```diff
@@ -0,0 +1,600 @@
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <services>
+                <service id="Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler" public="false">
+                    <argument>env(DATABASE_URL)</argument>
+
+                    <!-- you can also use PDO configuration, but requires passing two arguments: -->
+                    <!-- <argument>mysql:dbname=mydatabase, host=myhost</argument>
+                        <argument type="collection">
+                            <argument key="db_username">myuser</argument>
+                            <argument key="db_password">mypassword</argument>
+                        </argument> -->
+                </service>
+            </services>
+        </container>
+
+    .. code-block:: php
+
+        // config/services.php
+        use Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler;
+
+        $storageDefinition = $container->autowire(PdoSessionHandler::class)
+            ->setArguments([
+                '%env(DATABASE_URL)%',
+                // you can also use PDO configuration, but requires passing two arguments:
+                // 'mysql:dbname=mydatabase; host=myhost; port=myport',
+                // ['db_username' => 'myuser', 'db_password' => 'mypassword'],
+            ])
+        ;
+
+Next, use the :ref:`handler_id <config-framework-session-handler-id>`
+configuration option to tell Symfony to use this service as the session handler:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/framework.yaml
+        framework:
+            session:
+                # ...
+                handler_id: Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler
+
+    .. code-block:: xml
+
+        <!-- config/packages/framework.xml -->
+        <framework:config>
+            <!-- ... -->
+            <framework:session
+                handler-id="Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler"/>
+        </framework:config>
+
+    .. code-block:: php
+
+        // config/packages/framework.php
+        use Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler;
+
+        // ...
+        $container->loadFromExtension('framework', [
+            // ...
+            'session' => [
+                'handler_id' => PdoSessionHandler::class,
+            ],
+        ]);
+
+.. caution::
+
+    If the session data doesn't fit in the data column, it might get truncated
+    by the database engine. To make matters worse, when the session data gets
+    corrupted, PHP ignores the data without giving a warning.
+
+    If the application stores large amounts of session data, this problem can
+    be solved by increasing the column size (use ``BLOB`` or even ``MEDIUMBLOB``).
```

Comment:
PdoSessionHandler is already using `BLOB` when using its `createTable` method, so I don't understand this note.